### PR TITLE
Add repl commands mempeek and mempoke

### PR DESCRIPTION
**Pull request by chick**
Would be nice to have block versions of these but at least these give access.
One could construct script files to load blocks of memories using mempoke

### CODE REVIEW

File: src/main/scala/firrtl_interpreter/InterpretiveTester.scala [hunk 1]
```diff
@@ -122,17 +122,17 @@ class InterpretiveTester(input: String, optionsManager: HasInterpreterSuite = ne
     * Pokes value to the named memory at offset
     *
     * @param name  the name of a memory
-    * @param offset the offset in the memory
+    * @param index the offset in the memory
```
Comment (ucbjrl) [inline src/main/scala/firrtl_interpreter/InterpretiveTester.scala]: Why the change from `offset` to `index`? Won't this break existing code that uses named parameters? What are the units (bytes, words, ...)?

Comment (ucbjrl) [reply src/main/scala/firrtl_interpreter/InterpretiveTester.scala]: Actually, not requesting changes so much as clarification.

Comment (chick) [reply src/main/scala/firrtl_interpreter/InterpretiveTester.scala]: I just introduced the offset/index param in the previous PR, offset comes from the PeekPoke testers pokeAt where it makes more sense for a generic poke, but less sense than index for a memPoke.  I prefer the index, unless you really prefer offset.

Comment (ucbjrl) [reply src/main/scala/firrtl_interpreter/InterpretiveTester.scala]: Index is fine.
